docker/gaffer-ui/README.md

```diff
@@ -0,0 +1,4 @@
+Gaffer UI
+=======================
+
+The Gaffer UI image houses the Gaffer UI. The UI is based on 
```

Comment:
Is the end of this sentence missing?

Reply:
Lol yeah looks like I got halfway through documenting it and got bored 😆. I'll finish that off